fuse/all.py

```diff
@@ -322,4 +322,4 @@ def fuse_brady():
     missing_agency_df = find_event_agency_if_missing_from_post(event_df, post_event_df)
     post_event_df = post_event_df[~post_event_df["agency"].isin(event_df["agency"])]
     event_df = pd.concat([event_df, post_event_df], ignore_index=True)
-    event_df.to_csv(deba.data("fuse/event.csv"), index=False)
+    event_df.to_csv(deba.data("fuse/events_pre_post.csv"), index=False)
```

Comment:
Looks like typo here. Should be `fuse/event_pre_post.csv`. How was you able to run without issue on local?

Reply:
Thanks. I went back and forth between naming the table `events_pre_post` and `event_pre_post` so I have both on my local.